sbin/build.sh

```diff
@@ -303,6 +317,11 @@ configureVersionStringParameter() {
     BUILD_CONFIG[VENDOR_VERSION]="Bisheng"
     BUILD_CONFIG[VENDOR_BUG_URL]="https://gitee.com/openeuler/bishengjdk-${BUILD_CONFIG[OPENJDK_FEATURE_NUMBER]}/issues"
     BUILD_CONFIG[VENDOR_VM_BUG_URL]="https://gitee.com/openeuler/bishengjdk-${BUILD_CONFIG[OPENJDK_FEATURE_NUMBER]}/issues"
+  elif [[ "${BUILD_CONFIG[BUILD_VARIANT]}" == "${BUILD_VARIANT_LOONGSON}" ]]; then
+    BUILD_CONFIG[VENDOR]="Loongson"
+    BUILD_CONFIG[VENDOR_VERSION]="loongson"
+    BUILD_CONFIG[VENDOR_BUG_URL]=""
```

Comment:
Would prefer to have some valid URLs here else it's hard for users to know where to report bugs